src/Tribe/Timezones.php

```diff
@@ -279,10 +279,34 @@ public static function to_utc( $datetime, $tzstring, $format = null ) {
 		$new_datetime = date_create( $datetime, $local );
 
 		if ( $new_datetime ) {
-			$new_datetime->setTimezone( $utc );
-			$format = ! empty( $format ) ? $format : Tribe__Date_Utils::DBDATETIMEFORMAT;
-
-			return $new_datetime->format( $format );
+			if ( empty ( $format ) ) {
+				/**
+				 * In order to avoid problems with dates that fall into DST for local timeszones we need to split into: date & time
+				 * doing this we convert to UTC each in order to avoid a DST conversion based on date and time.
+				 *
+				 * This is important specially for recurring events as some of them might fall inside or outside of DST periods and as the conversion
+				 * might fall into timezones with DST PHP will automatically convert an hour from normal to DST or viceversa
+				 * in most cases the event that was supposed to be started at 12:00pm should remain at the same time
+				 * regardless if is DST or not.
+				 */
+				try {
+					$date = date_create( $new_datetime->format( Tribe__Date_Utils::DBDATEFORMAT ), $local );
+					$time = date_create( $new_datetime->format( Tribe__Date_Utils::DBTIMEFORMAT ), $local );
+					$date->setTimezone( $utc );
+					$time->setTimezone( $utc );
+					return sprintf(
+						'%s %s',
+						$date->format( Tribe__Date_Utils::DBDATEFORMAT ),
+						$time->format( Tribe__Date_Utils::DBTIMEFORMAT )
+					);
+				} catch( Exception $e ) {
+					error_log( 'There was an error while date_create() was called.' );
```

Comment:
I would avoid the `error_log()` we have notice classes on common that we can use if we want.

Reply:
Okay perfect I'm going to remove it ;)